New species: Gelaxi! (#642)
space-wizards/docs: src/en/space-station-14/character-species/proposals/gelaxi.md

---

#### Slippery fingers

When picking up items, Gelaxi will have a percentage chance to instantly drop the item (as if it were lubed). This percentage increases the more moist they are! This chance shouldn't be too large as to be a large determent to gameplay.

###### GAMEPLAY REASONING / IMPACT:
 - Punishes Gelaxi players for being too moist and not thinking about their moisture levels closely
 - This will probably make them slightly weaker against security specifically as after getting stunned, picking up your items would be harder.

---

#### Slime sliding

When walking over reagents, Gelaxi will move faster. This only occurs if they are over a certain moisture level. They also must shift and slow walk *out* of puddles, or they will trip and fall down. This is a minor speed boost, but should be fast enough to where they actively seek out going on reagents.

###### GAMEPLAY REASONING / IMPACT:
 - Gives Gelaxi a natural way to interact with the stations environment! It also isn't free to use, you have to be careful when leaving reagents, otherwise your liable to trip and ruin any advantage you had.

---

### Minor unique features

 - Gelaxi have very weak melee damage.

>  - Gelaxi are heavily poisoned by salt, it dries them out and does poison damage. This is only by ingestion or by walking over it - splashing it on them will do nothing so players can't instantly kill them.

Reviewer: I do not agree with the melee damage nerf, I understand that these are living snails and players should know the consequences of choosing to be a snail person, but with everything else on the doc they seem to be the weakest species with their unique abilities alone, and do not need microbalances.

| Designers | Implemented   | GitHub Links                                                                      |
|---|---------------|-----------------------------------------------------------------------------------|
| Ork | :x: No | ----- |

## Overview

> This is a proposal for the Gelaxi species. They are a gastropod species who wear shells and are slimey! They aren't particularly fast (in their talking, or walking!) and they take great pride in their shells and love to share, customize, and show them off. They are primarily inspired by FTL Slugs and Zoltans and, of course, Lady Cassandra but are quite distinct.

Reviewer 2: "Slimy" not "Slimey"

> This is a proposal for the Gelaxi species. They are a gastropod species who wear shells and are slimey! They aren't particularly fast (in their talking, or walking!) and they take great pride in their shells and love to share, customize, and show them off. They are primarily inspired by FTL Slugs and Zoltans and, of course, Lady Cassandra but are quite distinct.

Reviewer: ```suggestion
This is a proposal for the Gelaxi species. They are a gastropod species who wear shells and are slimy! They aren't particularly fast (in their talking, or walking!) and they take great pride in their shells and love to share, customize, and show them off. They are primarily inspired by FTL's Slugs and Zoltans, in addition to Lady Cassandra from Doctor Who, but are quite distinct.
```

## Overarching theme

> The species, in general, is focused on having a lot of interactions with the crew. Many of the Gelaxis abilities are concentrated on this aspect and its a big goal for the species as a whole. I think this is an aspect largly missing from other species and this is a good space to design in.

Reviewer 2: "largly"

### Visuals

> Gelaxi are just large snails, they themselves aren't very colorful, but they take great pride in customizing their shells which have many unique designs and color schemes. Shells should be fully customizable, before and during the round. You can affix objects to the shells (E.g papers or flashlights) and get them stamped or signed by various crew and command. They would also ideally have unique markings added depending on interactions that happen to them (E.g getting shot, lit on fire etc...). These shells are "living" art pieces, and ideally would change visually throughout the shift.

Reviewer 3: Slightly off-topic, but I would say that "mid-round interactions can lead to your character gaining new markings (e.g burn scars, gunshot wounds)" is a mechanic that, if added, should probably be added to all species, not just gelaxi; if a human gets their arms blown off and sown back on, they should probably have some visible sticthes!

Author: I agree! It would be a lot of fun

> Gelaxi are just large snails, they themselves aren't very colorful, but they take great pride in customizing their shells which have many unique designs and color schemes. Shells should be fully customizable, before and during the round. You can affix objects to the shells (E.g papers or flashlights) and get them stamped or signed by various crew and command. They would also ideally have unique markings added depending on interactions that happen to them (E.g getting shot, lit on fire etc...). These shells are "living" art pieces, and ideally would change visually throughout the shift.

Reviewer: Two questions here, first clarify on what you mean by they aren't very colorful, are their species colors clamped, similar to vox? My second request is for you to clarify further on what you mean by affixing objects, are you just taping a flashlight on top of them?

Author: Both of those are good questions, but I left it vague on purpose. I'm not sure:

1.) What is possible with color clamping?
2.) What more skilled artists think would look good?
3.) What is too much for a primarily LRP species?

Here are the ranges of visuals we could pick from:
1.) You can customize the shell with markings and colors in the loadout screen.
2.) You can do that, but also in round
3.) You can do that, but it also takes environmental damage
4.) You can sign the shell, add things to it (like taping a flashlight on it), stamping it etc....

I'm not sure what level we should aim for, I personally think 4 would be fun, but 3 might be better suited.

Reviewer: I think taping flashlights and other items would open up for annoying microbalance, stamping is the best option to me as it would be silly for a clown Gelaxi to have AA on their shell.

### Major unique features:

### Shells

Gelaxis shells are of vital importance to them. Shells themselves come in multiple shapes (see the Visuals section for more information on this aspect), each shape has slightly unique abilities.
 - Some of these could include:
   - Unique storage grids (Spirals, cones, etc...)
   - Different speed penalties.
   - Sound effects (Conch shell)
   - Reflect chance when being shot from the back (Trust me, we can balance it :))

>    - Various doafter lengths for unequiping the shells.

Reviewer 2: "unequiping" to "unequipping"

   - And more!

> Shells can be taken off without harm to the Gelaxis - in fact they *want* to do this! The longer a Gelaxi wears a shell, the slower they move in it, and longer it takes to open. This should take a long time (~50%+ of the shift). The only way to fix this is to swap their shell with another Gelaxi. Part of Gelaxis culture is to share shells so this is normal and expected of them. These downsides should be minor, you should be able to happily wear your shell without swapping it and not be affected in a large way.

Reviewer 3: Movement speed is an extremely important statistic for most roles in the game (most obviously stuff like security and antagonists, but essentially every role appreciates being able to just get around the station faster), so having a speed penalty probably isn't ideal (and, verisimilitude-wise, I'm not sure where this penalty would really be "coming from"). I like the concept of allowing players to swap shells, and I like the flavor you're aiming for here, but I think that making it a "swap or you face a gameplay penalty" mechanic isn't ideal because then the interaction goes from "pardon me, your shell is quite exquisite, do you have any tips for decorating mine?" to "oh hey, it's been like 30 minutes and we're both slow, let's just wordlessly swap shells and move on". This also means that if there's no other gelaxi, or the other gelaxi on the station are working jobs that can't easily cross paths with each other (e.g, one's an atmos tech and one's a bartender), then you kind of just need to eat the penalty. Based on what's written here, it also seems like there's no penalty for NOT wearing a shell, so a lot of players might just decide they don't care about the extra storage space and would rather just move faster.

Author: Couple of things:

I was not clear, but their shells replace their backpack so I don't see anyone ditching it for long periods of time.

What worries me about making swapping a non mechanical feature is that people will just never do it :cry:... Swapping stuff would take a decent amount of effort (not to mention if you have contraband or something). I would much much rather make it a buff of some sort, but the issue with that is there isn't a really good way of making that work. If you get a buff for swapping, then people will just do it round start which defeats the point.

I will say that the downsides are minor, I think if there are only two Galaxi on the station and they don't meet up, that's fine! The main issue I have is thinking of intresting downsides that aren't just boring is difficult. Maybe some kind of shader overlay (edges of your screen gets darker) because your feeling cramped?

If you have any more suggestions please let me know - this is one of those things that I'm not sure will work, but I think the flavor of swapping shells adds a lot so I'm trying to think of ways to make it more interesting.

> Shells can be taken off without harm to the Gelaxis - in fact they *want* to do this! The longer a Gelaxi wears a shell, the slower they move in it, and longer it takes to open. This should take a long time (~50%+ of the shift). The only way to fix this is to swap their shell with another Gelaxi. Part of Gelaxis culture is to share shells so this is normal and expected of them. These downsides should be minor, you should be able to happily wear your shell without swapping it and not be affected in a large way.

Reviewer: I am confused on what you mean by taken off, the way you've described shells makes them seem like they would replace the backpack slot, but this is never explained, please describe whether the shells are a unique slot or if they replace the backpack slot.

###### GAMEPLAY REASONING / IMPACT:
 - Players love large amounts of customization, shells being highly customizable gives them this freedom in a natural and fun way.
 - A unique feature that allows players to express themselves easily in multiple ways.
 - Gameplay wise, you can pick a shell that fits your playstyle (need a lot of storage but don't care about movement speed? Take the shell  that fits!).
 - This incentivises player on player interaction by giving gameplay reasons to swap shells.

---

### Moisture / slime

> Gelaxis are slimey creature and need moisture to be comfortable (but not too moist)! They can get moisture in various ways, but most, if not all, should be primarily be from other crew. For example, crew should be able to spray them with fire extinguishers and slimes should be able to hug them to increase their moisture level. Gelaxis *should not* be able to do this to themselves easily. Their current moisture level effects various aspects about them (movement speed, shell open doafter length etc...). See the other sections for how moisture effects it.

Reviewer 3: I like the concept of a moisture mechanic, but I think it shouldn't be overly inconvenient to the player. In my eyes, a drawback like this shouldn't be a major point of balance for the species so much as just a way to add verisimilitude (i.e, it shouldn't be "I need to drop everything I'm doing and go look for someone willing to make me moist" so much as "oh, right, I'm a weird slug alien; neat!").

It'd probably be good to compare vox here, since their oxygen allergy is in the same boat of "species-specific drawback that needs to be managed". A vox player doesn't generally struggle too much to manage their oxygen allergy by themselves, since the station has a lot of nitrogen tanks/canisters in maints, but they can benefit from cargo buying them a liquid nitrogen canister or atmospherics building an oxygen-free seating area in the bar. That is to say, I think moisture for gelaxi should follow a similar philosophy of being fairly easy for most players to manage on their own, but benefiting from certain departments going out of their way to provide improved quality-of-life; e.g, maybe chemistry could produce some kind of ointment that preserves moisture for longer than regular water does, or science can research some kind of wearable water reservoir that moistens automatically.

I'll also note that, implementation-wise, it'd probably be good to make this moisture system flexible enough to apply to other species, e.g if we decide to add an amphibious/aquatic species in the future.

Author: I agree with you! I wasn't very clear here, but the downsides for not being moist should be minor and not that oppressive. You should want to be at high moisture (usually, but not everytime!) but not maintaining it should only effect you in a minor way.

#### Slimey hands

> Gelaxies have no fingerprints, their slimey hands do not have enough definition for them! However, when touching objects, doors or worn clothing, they do leave some slime. This slime will indicate the color of the Gelaxi that touched (The text color will match the Gelaxis color) and will last for a couple of minutes, the exact time depends on how moist they are.

Reviewer: This may clash in concept with Slimepeople species, if I were a player I would be confused on why the snail people have no fingerprints but the slime people do

Reviewer 4: I mean I'd imagine slime people have a stronger, skin-like external membrane that would be leaving the prints, compared to Gelaxies which covered in a mucus.

Author: > I mean I'd imagine slime people have a stronger, skin-like external membrane that would be leaving the prints, compared to Gelaxies which covered in a mucus.

This is also how I've always imaged slimes as well :laughing:

Reviewer: > > I mean I'd imagine slime people have a stronger, skin-like external membrane that would be leaving the prints, compared to Gelaxies which covered in a mucus.
> 
> This is also how I've always imaged slimes as well 😆

Well it’s more an issue of a lack of slime people doc than yours anyways

Author: Hitlinemoss had a similar concern, I personally think this is fine because I think of slimes as rubbery not slimey - see my comment above for more details

> Gelaxies have no fingerprints, their slimey hands do not have enough definition for them! However, when touching objects, doors or worn clothing, they do leave some slime. This slime will indicate the color of the Gelaxi that touched (The text color will match the Gelaxis color) and will last for a couple of minutes, the exact time depends on how moist they are.

Reviewer 3: I really like the concept of "this species doesn't leave 'fingerprints' but instead leaves some kind of other obvious forensic evidence"; it adds some interesting complexities to detective gameplay. That being said, having the slime fade away after only a couple minutes means that it's likely long gone by the time security actually finds the item, so it'd probably be better for it to stick around until cleaned off, like normal fingerprints; otherwise, this is very close to just being "species that leaves no fingerprints".

Slightly off-topic, but this kind of mechanic would also be really good for slimes, who are also slimey, come in multiple colors, and don't have clearly-defined fingerprints.

Author: > I really like the concept of "this species doesn't leave 'fingerprints' but instead leaves some kind of other obvious forensic evidence"; it adds some interesting complexities to detective gameplay. That being said, having the slime fade away after only a couple minutes means that it's likely long gone by the time security actually finds the item, so it'd probably be better for it to stick around until cleaned off, like normal fingerprints; otherwise, this is very close to just being "species that leaves no fingerprints".

The exact time needs to be figured out, the main issue is if the time is too long, then security will be finding their "prints" everywhere, and it would be basically useless for tracking them down. I'm not sure the exact time at all, but it cannot last for too long otherwise it becomes much too strong for the galexi.

> Slightly off-topic, but this kind of mechanic would also be really good for slimes, who are also slimey, come in multiple colors, and don't have clearly-defined fingerprints.

I replied in the other thread about this, but I (personally) think as slimes as being more rubbery than slimey! Slugs seem much more icky and sticky, slimes seem like bouncy balls xD

Reviewer 3: > The exact time needs to be figured out, the main issue is if the time is too long, then security will be finding their "prints" everywhere, and it would be basically useless for tracking them down.

I don't really think this is an issue, since they'd be leaving their "fingerprints" on the same kinds of items as anyone else (i.e, items they pick up and use); that is to say, fingerprints already end up basically everywhere, and the forensic puzzle is tracking down whose fingerprints are on items of interest (like a weapon or a stolen item).

###### GAMEPLAY REASONING / IMPACT:
 - This makes detectives / securities interactions with them unique, if the crime happened recently, finding the suspect should be much easier, but if the crime happened long in the past, there will be no trace. Its incentives more complex plans by rewarding them with no evidence, and also disincentives run and gun tactics by allowing easy tracking by even non detective officers.

---

#### Slippery fingers

> When picking up items, Gelaxi will have a percentage chance to instantly drop the item (as if it were lubed). This percentage increases the more moist they are! This chance shouldn't be too large as to be a large determent to gameplay.

Reviewer 3: I don't really think this is a very interesting downside; most of the time it's just a weird but ultimately meaningless inconvenience, and then sometimes it's an annoyance that just instantly ruins the game (e.g picking up a medicine jug and spilling all of it all over the floor, or picking up a slipped nukie's weapon and instantly dropping it instead of being able to fight back with it).

Author: Here are some alternatives I was thinking of:

1.) When you pick up items, they get a space lube equalivent added to them. The next non gelaxi to pick them up will have it slip like lube. You could also wipe it off with a towel

2.) Instead of it being random, its on a timer. E.g it only happens every minute, but if you don't wipe your hands before touching something and the cooldown is off, then you slip when picking an item up.

Reviewer: I think something that stops Gelaxi players from being too moist is great, but there could be better ways to go about it, maybe longer do-afters if they're too moist rather than items being dropped.

###### GAMEPLAY REASONING / IMPACT:

>  - Intresting and "lore" accurate ability

Reviewer 2: "Intresting"

 - Punishes Gelaxi players for being too moist and not thinking about their moisture levels closely
 - This will probably make them slightly weaker against security specifically as after getting stunned, picking up your items would be harder.

---

#### Slime sliding

> When walking over reagents, Gelaxi will move faster. This only occurs if they are over a certain moisture level. They also must shift and slow walk *out* of puddles, or they will trip and fall down. This is a minor speed boost, but should be fast enough to where they actively seek out going on reagents.

Reviewer: Them tripping and falling out of puddles if they don't slow walk is cute, but this may be hard to communicate to players, I would recommend just axeing the tripping part, Gelaxi seem weak enough with everything else.

Reviewer 5: Agreed with removing tripping, this will get annoyingly detrimental very quickly.

Author: I don't think it will be that detrimental! Its literally the same as normal puddles, but you just have to remember it at the end instead of throughout.

I agree, it is unintuitive, but I personally think it is worth trying out to see how it feels in game before passing judgement. I also not sold on it speeding them up as well, I think probably just not moving slowly is a big enough buff

> When walking over reagents, Gelaxi will move faster. This only occurs if they are over a certain moisture level. They also must shift and slow walk *out* of puddles, or they will trip and fall down. This is a minor speed boost, but should be fast enough to where they actively seek out going on reagents.

Reviewer 3: It might be better for this to be a simpler "not slowed down by puddles of reagents" mechanic, since I think as written here this is too complex for what is ultimately an extremely niche upside (and one that the janitor will inevitably be interfering with, at that).

###### GAMEPLAY REASONING / IMPACT:
 - Gives Gelaxi a natural way to interact with the stations environment! It also isn't free to use, you have to be careful when leaving reagents, otherwise your liable to trip and ruin any advantage you had.

---

> ### Minor uinque features

Reviewer 2: "uinque"

- Gelaxi have very weak melee damage.
- Gelaxi are heavily poisoned by salt, it dries them out and does poison damage. This is only by ingestion or by walking over it - splashing it on them will do nothing so players can't instantly kill them.
- Any reagent that has salt as one of its precursors, will also poison them (slightly, not a lot!).
- Gelaxi will occasionally drop slime puddles on the ground depending on how moist they are, these will *not* be reagent puddles, but markings like dirt and will be the same color as their skin. They are purely visual. The janitor will want to clean them up to make the station spotless, but it wont effect the crew in a harmful way.

> - Gelaxi talk slowly in a unique accent. This should be minor, and not very noticeable.

Reviewer: If you could clarify what you mean by talk slowly, like how players talk when they're drunk?

Reviewer 3: I'd imagine it'd be something like the accent that sloths use when cogni'd. As in they talk... really... slowly... and... use... ellipses... after... every... word.

Author: Yeah exactly!

> - Gelaxi have a base slower movement speed than other species.

Reviewer 3: Something I generally want to note here: it's fine for gelaxi to have a slower base speed than other species (it's a pretty sizable drawback, but one that the player is actively signing up for by picking the literal snail species), but having additional slowdown from their shells is probably going to outright cripple them. It might be best to just have the base speed penalty and leave it at that, rather than having a second speed penalty stacked on top of that. (Moisture having a speed penalty can remain, since that's more of a "status effect" akin to being hungry/thirsty, so it's easier to manage.)

In theory you could go the other direction of removing the _base_ penalty and only having speed penalties from shells, but that'd likely result in players just ditching the shells entirely if possible, which we probably don't want.

Author: > In theory you could go the other direction of removing the base penalty and only having speed penalties from shells, but that'd likely result in players just ditching the shells entirely if possible, which we probably don't want.

That might be a better way to go about it, I wasn't clear in the doc but they *cannot* wear backpacks, so ditching your shell is a big deal (you also  can't easily get a new one) and if you really need that extra movement speed then I think that's a fair tradeoff.

---

### Electricity subtheme - Optional addon

This is an optional subtheme that I think could add a lot to the species. However, because of how many things the species already has going on, might be too much. This should be added if many of the other abilities are decided to be undesirable. Here is what this addon would constitute:

>  - Gelaxis eyes would be somewhat powerful flashlights (between PDA and flashlight), you can toggle them on with an action and they would be the same color as your eye color.

Reviewer 2: I think the plural for the species should just be "Gelaxi"

 - Gelaxi have unique markings on their shells that glow in the dark.
 - Gelaxi have an ability, and when activated, they turn into a small generator able to power parts of the station for short periods of time (~2-3 minutes)
 - Gelaxis vomit charges battery items when it comes into contact.
 - Strength of these abilities would be tied to their moisture level.

>  - No shoes

Reviewer: How does having no shoes add to the electricity subtheme?

---

### Clashes with other species

> To note, there are many abilities that "make sense" to give to Gelaxi. For example, its not a far stretch to allow them to eat trash like Vox, or suck up regents like Diona. This is a dangerous road to travel, and all abilities are carefully made to *not* tread on the unique aspects of other species.

Reviewer 2: "regents" to "reagents"

### Visuals

Gelaxi are just large snails, they themselves aren't very colorful, but they take great pride in customizing their shells which have many unique designs and color schemes. Shells should be fully customizable, before and during the round. You can affix objects to the shells (E.g papers or flashlights) and get them stamped or signed by various crew and command. They would also ideally have unique markings added depending on interactions that happen to them (E.g getting shot, lit on fire etc...). These shells are "living" art pieces, and ideally would change visually throughout the shift.

### Major unique features:

### Shells

Gelaxis shells are of vital importance to them. They replace the backpack slot and Gelaxis cannot wear normal backpacks. Shells themselves come in multiple shapes (see the Visuals section for more information on this aspect), each shape has slightly unique abilities.
 - Some of these could include:
   - Unique storage grids (Spirals, cones, etc...)
   - Different speed penalties.
   - Sound effects (Conch shell)

>    - Reflect chance when being shot from the back, front would not reflect (Trust me, we can balance it :))

Reviewer 6: We had a PR that tried to introduce directional reflection before, but it had to be declined due to technical reasons. It will also very likely make bullet prediction much harder to implement, so I would rather skip this feature.

Author: > We had a PR that tried to introduce directional reflection before, but it had to be declined due to technical reasons. It will also very likely make bullet prediction much harder to implement, so I would rather skip this feature.

Its just an example of an ability! I think it sounds cool but if its not possible technology wise, then yeah we wont add it. To be clear: if it DOESNT work I will *not* make some half-assed version (e.g reflect on all angles etc...)

   - Various doafter lengths for unequipping the shells.
   - And more!

Shells can be taken off without harm to the Gelaxis - in fact they *want* to do this! The longer a Gelaxi wears a shell, the slower they move in it, and longer it takes to open. This should take a long time (~50%+ of the shift). The only way to fix this is to swap their shell with another Gelaxi. Part of Gelaxis culture is to share shells so this is normal and expected of them. These downsides should be minor, you should be able to happily wear your shell without swapping it and not be affected in a large way.

###### GAMEPLAY REASONING / IMPACT:
 - Players love large amounts of customization, shells being highly customizable gives them this freedom in a natural and fun way.
 - A unique feature that allows players to express themselves easily in multiple ways.
 - Gameplay wise, you can pick a shell that fits your playstyle (need a lot of storage but don't care about movement speed? Take the shell  that fits!).
 - This incentivises player on player interaction by giving gameplay reasons to swap shells.

---

### Moisture / slime

> Gelaxis are slimey creature and need moisture to be comfortable (but not too moist)! They can get moisture in various ways, but most, if not all, should be primarily be from other crew. For example, crew should be able to spray them with fire extinguishers and slimes should be able to hug them to increase their moisture level. Gelaxis *should not* be able to do this to themselves easily. Their current moisture level effects various aspects about them (movement speed, shell open doafter length etc...). See the other sections for how moisture effects it. On the whole, their moisture level should not have massive downsides to not maintaining. Being too dry or too moist should not make the game extremely frustrating, at worst, it should be mildly annoying.

Reviewer 6: What does "to be comfortable" mean in this context? We don't have a mood system.

Author: > What does "to be comfortable" mean in this context? We don't have a mood system.

Purely fluff wording on how it should feel gameplay wise! Look at some of the features below for examples of what I mean
